### PR TITLE
AKS Kube version upgrade and considerations

### LINKED ISSUE #448: AKS can't pull AKS service broker image via doc instructions
A [kubernetes issue](https://github.com/kubernetes/kubernetes/pull/66429) and/or [azure service broker issue](https://github.com/Azure/open-service-broker-azure/issues/700) is causing `open-service-broker-azure` installation to fail due to inability to pull the `osbapublicacr.azurecr.io/microsoft/azure-service-broker:v1.6.0` image. Upgrading the AKS kubernetes version to 1.11.9 will fix this issue, but does require some additional workarounds and considerations which should also be documented

### LINKED ISSUE #449: Upgrading kubernetes version for CAP on AKS is undocumented.
When an admin decides to upgrade their kubernetes version on an AKS+CAP deployment, there are some considerations they need to be aware of which should be documented.

- The admin will need to enable swapaccounting and reboot all VMs, which may take some time.
```
# Assumes AZ_RG_NAME is set to the value of the resource group which contains their AKS deployment:
export AZ_MC_RG_NAME=$(az group list -o table | grep MC_"$AZ_RG_NAME"_ | awk '{print $1}')
vmnodes=$(az vm list -g $AZ_MC_RG_NAME | jq -r '.[] | select (.tags.poolName | contains("node")) | .name')
for i in $vmnodes; do
   az vm run-command invoke -g $AZ_MC_RG_NAME -n $i --command-id RunShellScript \
     --scripts "sudo sed -i -r 's|^(GRUB_CMDLINE_LINUX_DEFAULT=)\"(.*.)\"|\1\"\2 swapaccount=1\"|' /etc/default/grub.d/50-cloudimg-settings.cfg"
   az vm run-command invoke -g $AZ_MC_RG_NAME -n $i --command-id RunShellScript --scripts "sudo update-grub"
done

for i in $vmnodes; do
   az vm restart -g $AZ_MC_RG_NAME -n $i
done
```

- The admin should also expect 15-60 minutes of additional CAP downtime as CAP components resume.

### PR DESCRIPTION
**Pull request by btat**
Resolves #449 
Resolves #448

### CODE REVIEW

File: xml/cap_depl_aks.xml [hunk 1]
```diff
@@ -904,4 +913,46 @@ awk '($2 ~ /basic/) { system("cf enable-service-access " $1 " -p " $2)}'
...
+ <sect1 xml:id="sec.cap.aks-kubernetes-upgrade">
+  <title>Upgrading An AKS Cluster and Additional Considerations</title>
+
+  <para>
+   When upgrading the &kube; version of your AKS cluster, be sure to enable swap
+   accounting as it is required by &cap;. As part of the initial AKS cluster creation
+   process earlier in this chapter, swap accounting was enabled on all nodes. During the
+   AKS cluster upgrade process, these existing nodes are deleted and replaced by new nodes,
+   running a newer version of &kube;, but without swap accounting enabled. The procedure
+   below describes how to enable swap accounting on the new nodes added during a upgrade.
```
Comment (cmarin21) [inline xml/cap_depl_aks.xml]:
```suggestion
   below describes how to enable swap accounting on the new nodes added during an upgrade.
```

File: xml/cap_depl_aks.xml [hunk 2]
```diff
@@ -904,4 +913,46 @@ awk '($2 ~ /basic/) { system("cf enable-service-access " $1 " -p " $2)}'
...
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Upgrade your AKS cluster by following the process described in
+     <link xlink:href="https://docs.microsoft.com/en-us/azure/aks/upgrade-cluster"/>.
+    </para>
+    <para>
+     Modify the GRUB configuration of each node to enable swap accounting and then
+     reboot all nodes. <literal>RGNAME</literal> and <literal>AKSNAME</literal> will
```
Comment (cmarin21) [inline xml/cap_depl_aks.xml]:
```suggestion
     reboot all nodes. <literal>RGNAME</literal> and <literal>AKSNAME</literal> are the values ...
```

Comment (btat) [reply xml/cap_depl_aks.xml]: Resolved in https://github.com/SUSE/doc-cap/pull/460/commits/07ab36972c4ab629b75720fbccca7036bdb09c88

File: xml/cap_depl_aks.xml (hunk 1 above)
Comment (btat) [reply xml/cap_depl_aks.xml]: Resolved in https://github.com/SUSE/doc-cap/pull/460/commits/07ab36972c4ab629b75720fbccca7036bdb09c88

**Comment by btat** [comment]
> Billy,
> I commented on some small changes that are required. Other than that, it looks fine to me.

Thanks!

**Pull request merged**